src/main/java/teammates/storage/entity/Instructor.java

```diff
@@ -205,10 +205,10 @@ public String getInstructorPrivilegesAsText() {
         if (instructorPrivilegesAsText == null) {
             return null;
         }
-        return instructorPrivilegesAsText.getValue();
+        return instructorPrivilegesAsText.get();
     }
 
     public void setInstructorPrivilegeAsText(String instructorPrivilegesAsText) {
-        this.instructorPrivilegesAsText = new Text(instructorPrivilegesAsText);
+        this.instructorPrivilegesAsText = new StringValue(instructorPrivilegesAsText);
```

Comment:
Probably you need a null check here?

e.g.
```java
if (instructorPrivilegesAsText == null) {
    this.instructorPrivilegesAsText = null;
} else {
    this.instructorPrivilegesAsText = new StringValue(instructorPrivilegesAsText);
}
```

Reply:
Right, `StringValue` unlike `Text` does not support null value. I was thinking can construct `StringValue` using an empty string instead whenever `instructorPrivilegesAsText` is null, but this still results in many test failures. Also considering the use of `String` type as suggested by @wkurniawan07 without indexing, perhaps that would be a better solution.